File: java/parser/src/main/java/gov/lbl/parser/domain/Stored_definition.java
package gov.lbl.parser.domain;

import java.util.Collection;

public class Stored_definition {
    private String within;
    private Collection<Final_class_definition> final_class_definitions;

    public Stored_definition(String within, Collection<Final_class_definition> final_class_definitions) {
      this.within = within;
      this.final_class_definitions = final_class_definitions;
    }
}

File: jsParser/domain/Stored_definition.js
class Stored_definition {
  constructor (within, final_class_definitions) {
    within != null && within != '' ? this.within = within : ''
    final_class_definitions != null ? final_class_definitions.length != 0 ? this.final_class_definitions = final_class_definitions : '' : ''
  }
}
exports.Stored_definition = Stored_definition
